src/exercises/exercise04/Heading.js

```diff
@@ -1,25 +1,32 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Box } from 'grid-styled';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
+
+const headingStyles = css`
+	font-family: ${props => props.theme.fontFamily.heading};
+	font-size: ${props => props.theme.fontSize[props.size]};
+	color: ${props => props.theme.color.base};
+	font-weight: normal;
+`;
 
 /**
  * A text heading.
  */
 const Heading = () => {
-	return <div>Your code here</div>;
+	return <div className={headingStyles}>Your code here</div>;
 };
 
 Heading.propTypes = {
 	/** Custom component or HTML tag */
 	is: PropTypes.oneOfType([PropTypes.element, PropTypes.string]),
-	size: PropTypes.oneOf(['alpha', 'beta', 'gamma', 'delta', 'epsilon', 'zeta'])
-		.isRequired,
+	size: PropTypes.oneOf(['xxlarge', 'xlarge', 'large', 'base', 'small']),
```

Comment:
I'd suggest we go with `xxl | xl | lg | base | sm | xs | xxs`

Reply:
That was your idea ;-) But I'll change to `xxl` — like them more too ;-)

Reply:
But: `base` or `m`?

Reply:
Let's keep `base` for consistency with other tokens.